core/src/test/java/org/elasticsearch/http/netty/pipelining/HttpPipeliningHandlerTests.java

```diff
@@ -118,7 +118,8 @@ public ChannelPipeline getPipeline() throws Exception {
             }
         });
 
-        serverBootstrap.bind(HOST_ADDR);
+        Channel channel = serverBootstrap.bind(new InetSocketAddress(InetAddress.getLoopbackAddress(), 0));
```

Comment:
Nevermind, i see the serverBootstrap.shutdown() now. (it was hidden in the PR view)